webpage/scripts/generate_readme.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
```

Comment:
Should we remove this file from the current PR?

Reply:
I will remove last commit from this PR